### ISSUE TITLE
Execute button and Command Line Interface

### ISSUE THREAD

**Issue by michelevioglio97** [elsa-workflows/elsa-core#4559]
Don't you think that a "Run" button to start a workflow instance would be useful? It could be placed in the workflow definitions grid and/or directly in the settings tab inside the workflow designer. In the same tab, it could be put a CLI to see the output from WriteLine or read some input from ReadLine.

**Comment by sfmskywalker** [comment]
Definitely. the Elsa 3 designer has a Run button built-in. The console output would be a great addition to have 👍🏻
